tests/unit/restapi/v1/conftest.py

```diff
@@ -90,3 +90,29 @@ def registered_queues(
         "queue2": queue2_response,
         "queue3": queue3_response,
     }
+
+
+@pytest.fixture
+def registered_tags(
+    client: FlaskClient, db: SQLAlchemy, auth_account: dict[str, Any]
+) -> dict[str, Any]:
+    tag1_response = actions.register_tag(
+        client,
+        name="ML: Object Recognition",
```

Comment:
@jkglasbrenner Did we want to limit what characters can be used in tag names? I was thinking what I would typically think of as a tag would be something like `[a-zA-Z0-9_\-]`. I see it's just `Text` in the DB.

Reply:
@keithmanville Since we're borrowing the tag concept from GitHub's labels, I just attempted to make a label using spaces and special characters, looks like it works:

![image](https://github.com/usnistgov/dioptra/assets/8547754/6845466c-506f-48e0-9c4e-ce3d0cb3630d)

So I'm okay with not being restrictive on the tag names. Users/Groups always have the option to set their own agreed-upon rules for how they will make tags.